### PR TITLE
Adding doc for the publish robot

### PR DESCRIPTION
**Pull request by caesarxuchao**
@rmmh here is the doc. It's still rough but should be enough to get you start.

I also included some useful scripts.

cc @sttts

### CODE REVIEW

**Comment by rmmh** [comment]
Thanks for the documentation!

**Comment by sttts** [comment]
:+1:

File: mungegithub/publisher/README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+Then you can deploy the robot for real,
+
+* Change `config.organization` to "kubernetes" in `mungegithub/publisher/deployment/kubernetes/configmap.yaml`
+
+* Deploy the publishing robot by running [deploy.sh](util/deploy.sh)
+
+## Known issues
+
+1. Reporting issues: the publishing robot should file an issue and attach its logs if it meets bugs during publishing.
```
Comment (sttts) [inline mungegithub/publisher/README.md]: did this ever work? At least I cannot remember having got a notification of such an issue ever.

File: mungegithub/publisher/util/fetch-all-latest-and-push.sh [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+repo_count=${#repos[@]}
+for (( i=0; i<${repo_count}; i++ )); do
+    cd $K1/../"${repos[i]}"
+    echo "repo=${repos[i]}"
+    git branch -f release-1.7 upstream/release-1.7
+    git push -f origin release-1.7
+done
+
+echo "=================="
+echo "sync client-go release-3.0"
```
Comment (sttts) [inline mungegithub/publisher/util/fetch-all-latest-and-push.sh]: no 4.0 here?

File: mungegithub/publisher/README.md (hunk 1 above)
Comment (caesarxuchao) [inline mungegithub/publisher/README.md]: No. I once had a commit doing that, but the robot had more serious problems at that time so we didn't follow though.

**Pull request merged**